hadoop-ozone/dist/src/main/k8s/examples/ozone/datanode-statefulset.yaml

```diff
@@ -27,7 +27,6 @@ spec:
       component: datanode
   serviceName: datanode
   replicas: 3
-  podManagementPolicy: Parallel
```

Comment:
why was this removed?

Reply:
Nice catch.

`podManagementPolicy` and `imagePullPolicy` were added in 2ffbfff0c199898444c2ea3163e782b1dd9832e3 manually.  Such changes are confusing: example with manual edits does not match what we test.  We need to add these in the flekszible definitions instead, and regenerate the examples.

Reply:
On second thought, I'm not sure we should add these in the examples.  In our test environment it does not make a difference.  Users may add it in their own environment as needed.